doc/releasenotes/14_0_0_summary.md

```diff
@@ -26,6 +27,11 @@ You can always roll back to the earlier planner, either by providing the flag `-
 select /*vt+ PLANNER=V3 */ name, count(*) from users
 ```
 
+### Schema Tracking is no longer experimental
+
+Schema tracking is now GA. It is enabled by default. Schema tracking allows the Gen4 planner to plan many more queries.
+This can be turned off if necessary (for example, if you choose to continue to use the V3 planner) by changing the [relevant flags](https://vitess.io/docs/14.0/reference/features/schema-tracking/) on VTGate and VTTablet.
```

Comment:
schema tracking is still useful if you use v3 planner. v3 also takes advantage of the authoritative column list.

The reason someone wants to disable is when they see issues with it.

Reply:
Do you have examples of situations where v3 takes advantage of the authoritative column list? One that I can think of is if someone uses the `v3` planner but we discretely plan the query using `Gen4` if for instance there is a LEFT JOIN.

Reply:
@harshit-gangal, I have removed the mention of `v3` in the sentence.